src/Sentry/Protocol/Context/Trace.cs

```diff
@@ -23,28 +23,48 @@ public class Trace : ISpanContext, IJsonSerializable
         public SentryId TraceId { get; set; }
 
         /// <inheritdoc />
-        public string Operation { get; set; } = "unknown";
+        public string Operation { get; set; } = "";
```

Comment:
Ideally this should be nullable, but it shares the same interface `ISpanContext` as `ISpan` and `ITransaction` :/